src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -184,7 +184,7 @@
             </trans-unit>
             <trans-unit id="49">
                 <source>The file was only partially uploaded.</source>
-                <target>Het bestand is niet geheel geüpload.</target>
```

Comment:
But that's present tense right, where english is written in past tense. :confused: im not a 
language purist though, nor is google ;-)